prowler/lib/check/compliance_models.py

```diff
@@ -129,8 +129,33 @@ class Mitre_Requirement_Attribute(BaseModel):
     Comment: str
 
 
+# MITRE Requirement Attribute
+class Mitre_Requirement_Attribute_Azure(BaseModel):
+    """MITRE Requirement Attribute"""
+
+    AzureService: str
+    Category: str
+    Value: str
+    Comment: str
+
+
+# MITRE Requirement
+class Mitre_Requirement_AWS(BaseModel):
+    """Mitre_Requirement holds the model for every MITRE requirement"""
+
+    Name: str
+    Id: str
+    Tactics: list[str]
+    SubTechniques: list[str]
+    Description: str
+    Platforms: list[str]
+    TechniqueURL: str
+    Attributes: list[Mitre_Requirement_Attribute_AWS]
+    Checks: list[str]
```

Comment:
I think it would be better to leave it as it was: `Mitre_Requirement` and do an Union of the different Attributes.

Reply:
Could make sense yeah, I'll check it